backend/backend.go

```diff
@@ -0,0 +1,59 @@
+package main
+
+import (
+	"log"
+	"net/http"
+	"path/filepath"
+	"strconv"
+
+	"gopkg.in/ini.v1"
+
+	"github.com/ucsdscheduleplanner/UCSD-Schedule-Planner/backend/db"
+	"github.com/ucsdscheduleplanner/UCSD-Schedule-Planner/backend/routes"
+)
+
+// TODO: make this config-able
+const port = 8080
+
+// create closure for http handler func
```

Comment:
so for comments like this, I can see that it's creating a closure, but why exactly do you need a closure? Is it to persist the database struct object for all function calls?

Reply:
Yes